.github/workflows/very-expensive-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Very Expensive Test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      # WARN: This will run very expensive test every time ANY label is added
+      - labeled
+  schedule:
+    - cron: '0 0 * * *' # Runs nightly at 0AM UTC
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  test:
+    name: Test
+    if: github.event_name == 'schedule' || contains(github.event.pull_request.labels.*.name, 'need/very-expensive-tests')
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      run_very_expensive_tests: true
+  issue:
+    name: Issue
+    if: failure() && github.event_name == 'schedule'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create issue
```

Comment:
And what happens if it keeps failing each day?  Will additional comments be added?  Basically it would be nice to see if this is still continuing to occur or if it was a one-off.

Reply:
As for now, it creates an issue per failure. We can certainly modify it to comment instead if an issue exists already and is open. Would that work?

Reply:
> We can certainly modify it to comment instead if an issue exists already and is open. Would that work?

Yeah, lets do that.  I think that will be preferred by maintainers.